README.md

```diff
@@ -7,8 +7,7 @@ Scala grammar for [tree-sitter](https://github.com/tree-sitter/tree-sitter).
 
 References
 
-* [The Scala Language Specification](http://www.scala-lang.org/docu/files/ScalaReference.pdf)
```

Comment:
I'm not sure what happened to this, but it's not there anymore and I couldn't easily find what replaced it. For now I figured it's best to just remove it.